hardware_interface/src/component_parser.cpp

```diff
@@ -211,6 +213,20 @@ std::string parse_data_type_attribute(const tinyxml2::XMLElement * elem)
   return data_type;
 }
 
+/// Parse is_async attribute
+/**
+ * Parses an XMLElement and returns the value of the is_async attribute.
+ * Defaults to "false" if not specified.
+ *
+ * \param[in] elem XMLElement that has the data_type attribute.
+ * \return boolean specifying the if the value read was true or false.
+ */
+bool parse_is_async_attribute(const tinyxml2::XMLElement * elem)
+{
+  const tinyxml2::XMLAttribute * attr = elem->FindAttribute(kIsAsyncAttribute);
+  return attr ? strcmp(attr->Value(), "true") == 0 : false;
```

Comment:
a little fool-proofing: I think it'd be worth also checking for `True` for all those who grew up with Python

Reply:
I've decided to use strcasecmp instead.

Reply:
Nice, thanks!